src/component/handlers/edit/editOnKeyDown.js

```diff
@@ -101,6 +107,19 @@ function editOnKeyDown(e: SyntheticKeyboardEvent): void {
     case Keys.DOWN:
       this.props.onDownArrow && this.props.onDownArrow(e);
       return;
+    case Keys.SPACE:
+      // handling for OSX where option + space scrolls
+      if (isChrome && isOptionKeyCommand(e)) {
+        e.preventDefault();
+        // insert a space into the editor
+        const contentState = DraftModifier.replaceText(
+          editorState.getCurrentContent(),
+          editorState.getSelection(),
+          ' '
```

Comment:
Is this the correct character to insert here, if the goal of <kbd>Opt</kbd>+<kbd>Space</kbd> is to insert a non-breaking space? Or should this be `'\u00a0'`?